common/utils.c

```diff
@@ -7,6 +7,7 @@
  */
 
 #include "includes.h"
+#include <ftw.h>
```

Comment:
we should have #include <ftw.h> before to #include "includes.h"

Reply:
Done